Blimp/Parameters.h

```diff
@@ -148,7 +148,6 @@ class Parameters
         // 140: Sensor parameters
         //
         k_param_compass,
-        k_param_frame_type,
```

Comment:
these `k_param_` should be re-named `k_param_unused` or similar. Stops us reusing the index and loading the old param value.

Reply:
I hope its correct now...first time playing with param removal